cpp/ps0006/ps0006_solv.cpp

```diff
@@ -47,7 +47,7 @@ int main()
     cout<<"\n------------------------------------- \n";

```

Comment:
- [x] counter gives error message for invalid value of `step`
- [x] up counter gives error message for negative value of  `step`  
- [x] down counter gives error message for positive value of  `step`
- [x] counter gives error message to count from a number to the same number unless `step==0`